src/webui/abstractwebapplication.cpp

```diff
@@ -366,9 +367,9 @@ bool AbstractWebApplication::sessionStart()
 bool AbstractWebApplication::sessionEnd()
 {
     if ((session_ != 0) && (sessions_.contains(session_->id))) {
-        QNetworkCookie cookie(C_SID, session_->id.toUtf8());
```

Comment:
>Is name only enough to remove cookie?

Yes.

>Is cookies are not identified by name=value pair?

Actually you can think it acts like std::map, 1 key maps to 1 value, so just having the key should be enough.

My original intention was setting the `value` to an invalid value as recommended by OWASP.
https://www.owasp.org/index.php/Session_Management_Cheat_Sheet#Session_Expiration